[2024-08-02T11:32:15.886+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-02T11:32:15.895+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.create_dim_user_type scheduled__2024-08-02T10:30:00+00:00 [queued]>
[2024-08-02T11:32:15.899+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.create_dim_user_type scheduled__2024-08-02T10:30:00+00:00 [queued]>
[2024-08-02T11:32:15.899+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-08-02T11:32:15.906+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): create_dim_user_type> on 2024-08-02 10:30:00+00:00
[2024-08-02T11:32:15.911+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=1485) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-02T11:32:15.912+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'create_dim_user_type', 'scheduled__2024-08-02T10:30:00+00:00', '--job-id', '320', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpksou19n5']
[2024-08-02T11:32:15.913+0000] {standard_task_runner.py:91} INFO - Job 320: Subtask create_dim_user_type
[2024-08-02T11:32:15.913+0000] {standard_task_runner.py:64} INFO - Started process 1492 to run task
[2024-08-02T11:32:15.961+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.create_dim_user_type scheduled__2024-08-02T10:30:00+00:00 [running]> on host 69633e19c226
[2024-08-02T11:32:16.015+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Gabriel Almeida' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='create_dim_user_type' AIRFLOW_CTX_EXECUTION_DATE='2024-08-02T10:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-02T10:30:00+00:00'
[2024-08-02T11:32:16.017+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-02T11:32:16.017+0000] {sql.py:282} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS DimUserType (
                id_user_type INT PRIMARY KEY,
                user_type VARCHAR(255)
            );
        
[2024-08-02T11:32:16.023+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-08-02T11:32:16.029+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-08-02T11:32:16.042+0000] {sql.py:495} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS DimUserType (
                id_user_type INT PRIMARY KEY,
                user_type VARCHAR(255)
            );
        , parameters: None
[2024-08-02T11:32:16.045+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-02T11:32:16.056+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=create_dim_user_type, run_id=scheduled__2024-08-02T10:30:00+00:00, execution_date=20240802T103000, start_date=20240802T113215, end_date=20240802T113216
[2024-08-02T11:32:16.098+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-02T11:32:16.106+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
